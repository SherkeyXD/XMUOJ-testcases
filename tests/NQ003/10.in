90
54 501 561 718
344 148 694 950
336 872 944 565
745 333 788 436
428 667 264 165
752 110 880 860
875 688 300 156
517 260 952 520
918 560 400 892
884 920 10 797
285 234 72 568
460 594 500 474
792 385 713 705
628 383 440 979
732 320 630 75
865 465 265 401
810 10 30 714
834 200 644 78
825 414 544 846
736 710 228 260
252 489 400 678
100 441 772 666
120 424 89 875
16 744 520 500
648 828 66 551
710 204 978 854
578 50 412 451
603 54 686 200
300 770 568 382
488 312 595 556
240 88 880 160
526 601 616 512
120 732 409 510
904 10 302 192
504 620 339 820
488 579 680 200
200 774 175 850
90 88 575 117
800 382 440 679
584 532 420 750
108 940 800 150
925 574 216 872
850 544 119 177
960 200 623 86
416 325 782 606
824 900 211 656
360 420 788 304
675 840 74 869
387 456 380 465
120 608 587 352
447 617 16 379
950 176 800 777
844 560 463 750
654 200 962 920
379 660 956 840
195 718 704 192
419 825 630 835
184 420 109 551
328 954 760 58
235 937 31 120
984 891 910 685
483 880 136 144
752 429 742 914
532 380 365 352
367 284 210 705
20 162 465 72
669 578 772 452
864 594 234 549
612 258 218 929
949 800 910 88
847 524 733 36
978 875 481 260
474 398 278 536
654 493 814 94
936 70 586 108
395 830 372 980
451 502 886 88
720 296 710 528
884 330 465 650
260 313 669 843
448 325 280 436
628 860 932 388
150 54 458 102
500 471 530 855
708 210 634 722
853 600 238 241
948 975 624 308
416 600 456 192
574 501 968 16
464 542 599 928
