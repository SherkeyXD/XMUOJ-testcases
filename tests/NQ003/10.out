551.49
875.05
681.11
111.62
528.11
760.84
783.36
506.78
615.26
882.61
396.14
126.49
329.61
624.95
265.38
603.40
1050.72
225.80
515.35
678.65
240.05
708.67
452.06
559.96
644.56
703.08
434.00
167.94
471.56
266.43
644.04
126.57
364.42
628.91
259.28
424.86
80.01
485.87
466.70
272.80
1050.22
769.08
817.95
355.76
461.43
659.78
443.44
601.70
11.40
532.56
492.35
619.44
425.75
783.11
604.42
731.95
211.24
150.95
994.71
842.08
218.89
813.70
485.10
169.33
449.32
454.01
162.74
631.61
778.12
713.07
501.14
790.72
239.71
429.88
352.06
151.75
600.52
232.22
527.22
669.46
201.36
561.43
311.72
385.17
517.32
712.11
741.53
409.96
624.87
408.93
